uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/AttachmentsPopupElementView.qml

```diff
@@ -214,6 +214,7 @@ ColumnLayout {
                     id: fetchAttachment
                     running: model.listModelData.fetchingAttachment
                     visible: model.listModelData.fetchingAttachment
+                    Layout.maximumHeight: fileInfoColumn.height
```

Comment:
I did a quick test of a few different styles and noticed that the styling size for the busy indicator changed its size dramatically which would alter the grid layout row size but would not go back down dynamically. Setting the maximum size on this ensures consistency regardless of style. If the icon is small it fits just fine but it won't get overly large and cause distortion of the grid layout now. Low risk as it just set's an upper limit that's used elsewhere on elements in this grid layout.